qrcode doc correction

ankitdas13 opened the pull request:
Doc reference : https://razorpay.com/docs/api/qr-codes

sofiya02 reviewed (changes requested):
[Close a QR Code](https://razorpay.com/docs/api/qr-codes/#close-a-qr-code) Remove semi-colon 

[Fetch a QR Code](https://razorpay.com/docs/api/qr-codes/#fetch-a-qr-code) -  "on_demand" missing in response

[Fetch QR Code for a Payment ID](https://razorpay.com/docs/api/qr-codes/#fetch-qr-code-for-a-payment-id) - "on_demand" missing in response

[Fetch Payments for a QR Code](https://razorpay.com/docs/api/qr-codes/#fetch-payments-for-a-qr-code) - update response

[Refund a Payment](https://razorpay.com/docs/api/qr-codes/#refund-a-payment) - please add refund api

ankitdas13 commented:
@sofiya02 
[Fetch a QR Code](https://razorpay.com/docs/api/qr-codes/#fetch-a-qr-code) - "on_demand" missing in response
added [here](https://github.com/razorpay/razorpay-node/blob/qr_doc_correction/documents/qrcode.md?plain=1#L221)

[Fetch QR Code for a Payment ID](https://razorpay.com/docs/api/qr-codes/#fetch-qr-code-for-a-payment-id) - "on_demand" missing in response
added [here](https://github.com/razorpay/razorpay-node/blob/qr_doc_correction/documents/qrcode.md?plain=1#L304)

sofiya02 reviewed (approved):
LGTM
